README.md

```diff
@@ -1,2 +1,47 @@
+# ![Alt text](docs/tersetspng.jpg "TerseTS")
+
 # TerseTS
-TerseTS is a collection of lossless and lossy time series compression methods.
+TerseTS is a library that provides methods for lossless and lossy compressing time series. To match existing litterature the methods are organized based on [Time Series Compression Survey](https://dl.acm.org/doi/10.1145/3560814). The library is implemented in Zig and provides a C-API with [bindings](#Installation) for other languages.
+
+# Installation
+TerseTS can be compiled and cross-compiled from source:
+1. Install the latest version of [Zig](https://ziglang.org/)
+2. Build TerseTS for development in `Debug` mode using Zig, e.g.,:
+   - Linux: `zig build -Dtarget=x86_64-linux`
+   - macOS: `zig build -Dtarget=aarch64-macos`
+   - Microsoft Windows: `zig build -Dtarget=x86_64-windows`
+3. Build TerseTS for deployment in `ReleaseFast`, `ReleaseSafe`, and `ReleaseSmall` mode using Zig, e.g.,:
+   - Linux: `zig build -Dtarget=x86_64-linux`
```

Comment:
One of the release build types should part of the example.

Reply:
Fixed.